mscore/metaedit.cpp

```diff
@@ -44,7 +44,13 @@ MetaEditDialog::MetaEditDialog(Score* s, QWidget* parent)
             revision->setText(QString::number(rev, 16));
       else
             revision->setText(QString::number(rev, 10));
-      filePath->setText(score->importedFilePath());
+
+      QString currentFileName  = score->masterScore()->fileInfo()->absoluteFilePath();
+      QString previousFileName = score->importedFilePath();
+      if (previousFileName.isEmpty() || previousFileName == currentFileName) // New score or no "Save as" used
+            filePath->setText(previousFileName);
+      else
+            filePath->setText(QString("%1\n%2\n%3").arg(currentFileName, "initially read from:", previousFileName));
```

Comment:
How is this new string going to get translated?

Reply:
Fixed, noticed myself ;-)